dashboards/cloudfront-dashboard-templates/cloudfront_realtime_logs-analysis.yaml

```diff
@@ -0,0 +1,195 @@
+dashboards:
+  CLOUDFRONT_REALTIME_LOGS ANALYSIS:
+    name: cloudfront_realtime_logs analysis
```

Comment:
why analysis?

Reply:
this seems to have taken the name of analysis while exporting using cid-cmd. can i change it manually without breaking the template?

Reply:
yes

Reply:
updated both files.

Reply:
name is something that customer see as title 

CloudFront Realtime Logs

dahsbboard id is url so dashes are ok

BTW is it real RealTime?

Reply:
Realtime - Logs are realtime, dashboard refresh, the fastest is 15 mins based on dataset refresh time. Exploring ways to speed it up further.